ui/src/components/credentials/CredentialLinkQR.tsx

```diff
@@ -0,0 +1,310 @@
+        hasAsyncTaskFailed(credentialQRCheck)
+      ) {
+        clearInterval(checkQRCredentialStatusTimer);
+      } else {
+        void checkCredentialQRCode();
+      }
+    }, POLLING_INTERVAL);
+
+    return () => clearInterval(checkQRCredentialStatusTimer);
+  }, [shareCredentialQRCode, linkID, credentialQRCheck, env]);
+
+  const onStartAgain = () => {
+    makeRequestAbortable(createCredentialQR);
+    setCredentialQRCheck({ status: "pending" });
+  };
+
+  const hasFailed = hasAsyncTaskFailed(shareCredentialQRCode)
+    ? shareCredentialQRCode.error
+    : hasAsyncTaskFailed(credentialQRCheck)
+    ? credentialQRCheck.error
+    : undefined;
+
+  if (hasFailed) {
+    if (hasFailed.status === 404) {
+      return (
+        <Space align="center" direction="vertical" size="large">
+          <Avatar className="avatar-color-error" icon={<QRIcon />} size={56} />
+
+          <Typography.Title level={2}>Claim link is invalid</Typography.Title>
+
+          <Typography.Text type="secondary">
+            In case you think this is an error, please contact the issuer of this claim.
+          </Typography.Text>
+        </Space>
+      );
+    } else if (hasFailed.status === 400) {
+      return (
+        <Space align="center" direction="vertical" size="large">
+          <Avatar className="avatar-color-error" icon={<AlertIcon />} size={56} />
+
+          <Typography.Title level={2}>QR code expired, start again</Typography.Title>
+
+          <Button icon={<IconRefresh />} onClick={onStartAgain} type="link">
+            Start again
+          </Button>
+        </Space>
+      );
+    }
+    return (
+      <ErrorResult error={hasFailed.message} labelRetry="Start again" onRetry={onStartAgain} />
+    );
+  }
+
+  if (!isAsyncTaskDataAvailable(shareCredentialQRCode)) {
+    return <LoadingResult />;
+  }
+
+  if (isAsyncTaskDataAvailable(credentialQRCheck) && credentialQRCheck.data.status !== "pending") {
+    const { proofTypes } = shareCredentialQRCode.data.linkDetail;
+    if (proofTypes.length > 1) {
+      return (
+        <>
+          <Space align="center" direction="vertical" size="large">
+            <Avatar className="avatar-color-success" icon={<CheckIcon />} size={56} />
+
+            <Typography.Title level={2}>Credential sent via notification, but...</Typography.Title>
+
+            <Typography.Text style={{ fontSize: 18 }} type="secondary">
+              We will send you an updated version of this credential containing an on-chain proof.
+              <Typography.Paragraph style={{ fontSize: 18, textAlign: "center" }} type="secondary">
+                Please ensure that you have enabled push notifications on the application.
+              </Typography.Paragraph>
+            </Typography.Text>
+
+            <Space>
+              <Button onClick={() => setIsModalOpen(true)}>Missed the notification?</Button>
+              <Button icon={<IconRefresh />} onClick={onStartAgain}>
```

Comment:
There are no more start again buttons for these flows, only on errors (check designs).